docs/MongoDBInstallv4.md

```diff
@@ -79,13 +86,21 @@ This is achieved by creating a replica set
 ### 1. Edit your config file
 
 Edit the configuration file /etc/mongod.conf and add the following config
-You can choose any name to replace "opensilex"
+You can choose any name to replace "opensilex" in the /etc/mongod.conf file:
 
 ```
 replication:
   replSetName: "opensilex"
 ```
 
+To change the configuration file, you should access it with super-user privilege, using the text editor of you choice, for instance:
```

Comment:
It's too detailled, with "Edit the configuration file /etc/mongod.conf" sentence you have the information